### ISSUE TITLE
Questions about validation set?

### ISSUE THREAD

**Issue by Victory8858** [DIG-Beihang/PSN#4]
Thanks for your greate work! I find that you use trainval both for train and validate in your work, I want to know if it is fair to any others work? Is it need to split a test set inpendent of the train set?
